### PR TITLE
fix every fixable checkstyle error

### PR DESCRIPTION
**Pull request by NotAName320**

### CODE REVIEW

**Review by ArnavChandna — APPROVED** [review]
Looking good

File: newspeek/src/main/java/use_case/load_url/LoadURLOutputBoundary.java [hunk 1]
```diff
@@ -1,14 +1,17 @@
 package use_case.load_url;
 
+/**
+ * The Output Boundary Interface for the Load URL Use Case.
+ */
 public interface LoadURLOutputBoundary {
     /**
-     * Prepares the success view for the Login Use Case.
```
Comment (tjcaul) [inline newspeek/src/main/java/use_case/load_url/LoadURLOutputBoundary.java:5]: good catch

File: newspeek/src/main/java/view/ReaderView.java [hunk 2]
```diff
@@ -43,30 +50,29 @@ public class ReaderView extends JPanel implements PropertyChangeListener {
 
     // NewsPeek title
     private static final Color NEWSPEEK_COLOR = new Color(70, 130, 180);
-    private static final Font NEWSPEEK_TITLE_FONT = new Font("SansSerif", Font.BOLD, 36);
+    private static final Font NEWSPEEK_TITLE_FONT = new Font(Font.SANS_SERIF, Font.BOLD, 36);
```
Comment (tjcaul) [inline newspeek/src/main/java/view/ReaderView.java:53]: so much cleaner!

**Review by tjcaul — APPROVED** [review]
This makes the project so much easier to understand. Also, our documentation is much more up-to-date now with the lab-5 'login' stuff finally gone

**Pull request merged**